spring/src/main/java/org/axonframework/spring/authorization/UnauthorizedCommandException.java

```diff
@@ -0,0 +1,18 @@
+package org.axonframework.spring.authorization;
+
+/**
+ * Exception indicating that a command has been rejected due to a lack of authorization.
+ *
+ * @author Roald Bankras
+ */
+public class UnauthorizedCommandException extends RuntimeException {
```

Comment:
I think it would be nice if we have this exception implement the `AxonNonTransientException`. That way, it can play along in our retry mechanism, clearly stating the command from such an exception cannot be retried.